submission/persistence-filesystem/src/test/kotlin/ac/uk/ebi/biostd/persistence/fire/FireFilesServiceTest.kt

```diff
@@ -1,194 +0,0 @@
-package ac.uk.ebi.biostd.persistence.fire
-
-import ac.uk.ebi.biostd.persistence.filesystem.fire.FireFilesService
-import ac.uk.ebi.biostd.persistence.filesystem.request.FilePersistenceRequest
-import ac.uk.ebi.biostd.persistence.filesystem.service.FileProcessingService
-import arrow.core.Either.Companion.left
-import ebi.ac.uk.extended.model.ExtAttribute
-import ebi.ac.uk.extended.model.ExtFileType.DIR
-import ebi.ac.uk.extended.model.ExtFileType.FILE
-import ebi.ac.uk.extended.model.ExtSection
-import ebi.ac.uk.extended.model.ExtSubmission
-import ebi.ac.uk.extended.model.FireFile
-import ebi.ac.uk.extended.model.createNfsFile
-import ebi.ac.uk.io.ext.createDirectory
-import ebi.ac.uk.io.ext.md5
-import ebi.ac.uk.io.ext.newFile
-import ebi.ac.uk.test.basicExtSubmission
-import ebi.ac.uk.util.collections.ifLeft
-import io.github.glytching.junit.extension.folder.TemporaryFolder
-import io.github.glytching.junit.extension.folder.TemporaryFolderExtension
-import io.mockk.clearAllMocks
-import io.mockk.every
-import io.mockk.impl.annotations.MockK
-import io.mockk.junit5.MockKExtension
-import io.mockk.slot
-import io.mockk.verify
-import org.assertj.core.api.Assertions.assertThat
-import org.junit.jupiter.api.AfterEach
-import org.junit.jupiter.api.BeforeEach
-import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.extension.ExtendWith
-import uk.ac.ebi.extended.serialization.service.ExtSerializationService
-import uk.ac.ebi.fire.client.integration.web.FireWebClient
-import uk.ac.ebi.fire.client.model.FireApiFile
-import uk.ac.ebi.serialization.common.FilesResolver
-import java.io.File
-import kotlin.io.path.exists
-
-@ExtendWith(MockKExtension::class, TemporaryFolderExtension::class)
-class FireFilesServiceTest(
```

Comment:
I think this test is still relevant as a unit test for the FireFilesService, we should just adjust it accordingly